examples/consul/go-server/config/server.yml

```diff
@@ -0,0 +1,58 @@
+# dubbo server yaml configure file
+
+
+# application config
+application_config:
+  organization : "ikurento.com"
+  name : "BDTService"
+  module : "dubbogo user-info server"
+  version : "0.0.1"
+  owner : "ZX"
+  environment : "test"
+
+registries :
+  "hangzhouzk":
```

Comment:
use zk in consul example?

Reply:
@AlexStocks On yes, it's my mistake. I will change a name for consul registry.

Reply:
@AlexStocks I modify yaml in new commit.